releasenotes/notes/deprecate-bind-parameters-283c94069e8a9142.yaml

```
@@ -0,0 +1,6 @@
+---
+deprecations:
+  - |
+    The method :meth:`~qiskit.QuantumCircuit.bind_parameters` is now deprecated. Its functionality
```

```suggestion
    The method :meth:`.QuantumCircuit.bind_parameters` is now deprecated. Its functionality
```
In this case, it's slightly better for readers if we include the `QuantumCircuit` qualifier the first time we mention one of the methods so they'll know what we're talking about.

The other cases in this file are fine to strip it, though I'm not 100% sure they'll lookup correctly - `QuantumCircuit` is documented as `qiskit.circuit.QuantumCircuit`, not `qiskit.QuantumCircuit` (it's usually easiest to use the `.` prefix to do a suffix match so we don't need to worry about it; `` :meth:`~.QuantumCircuit.assign_parameters` `` will match what we want, and only display the `assign_parameters` bit).

We also typically highlight that "deprecated" means "this will be removed".